File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171704/Business/TProyecto.cs
using System.Collections.Generic;

namespace Business
{
    public class TProyecto
    {
	    private static Data.TProyecto obj = new Data.TProyecto();

        public static int Add(Models.TProyecto value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.TProyecto value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.TProyecto GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.TProyecto GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.TProyecto> GetAll()
        {
            return obj.GetAll();
        }
    }
}

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/TProyecto.vb
Imports System.Collections.Generic

Public Class TProyecto
    Public Shared Function Add(value As Models.TProyecto) As Integer
        Return Data.TProyecto.Add(value)
    End Function

    Public Shared Function Edit(value As Models.TProyecto) As Integer
        Return Data.TProyecto.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.TProyecto.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.TProyecto
        Return Data.TProyecto.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.TProyecto
        Return Data.TProyecto.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.TProyecto)
        Return Data.TProyecto.GetAll()
    End Function
End Class